client/my-sites/controller.js

```diff
@@ -312,14 +312,17 @@ export function siteSelection( context, next ) {
 
 	if ( currentUser && currentUser.site_count === 0 ) {
 		renderEmptySites( context );
-		return analytics.pageView.record( basePath, sitesPageTitleForAnalytics + ' > No Sites' );
+		return analytics.pageView.record( '/no-sites', sitesPageTitleForAnalytics + ' > No Sites', {
+			basePath: basePath,
```

Comment:
Shouldn't the property name be `base_path` to conform with Tracks conventions?

From the [analytics lib readme](https://github.com/Automattic/wp-calypso/tree/533402712b818b63c7d212a90776f8934fb646b3/client/lib/analytics#tracks-naming-conventions):
> Each token in the event and property names should be separated by an underscore (_), not spaces or dashes.

Reply:
Good catch! Updated in a99494eabc369c2850c61d1ee3121aae704f7cf4.